src/components/Navbar.js

```diff
@@ -1,17 +1,142 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import '../index.css';
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+import styled from 'styled-components';
+
+const Navbar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpen = () => {
+    setIsOpen(true);
+  };
+
+  const handleClose = () => {
+    setIsOpen(false);
+  };
 
-function NavBar() {
   return (
-    <div className="nav-bar">
-      <h1>Book Store</h1>
-      <div>
-        <Link to="/">Books</Link>
-        <Link to="/catagories">catagories</Link>
-      </div>
-    </div>
+    <Nav>
+      <MainTitle>Bookstore CMS</MainTitle>
+      {isOpen ? (
+        <CloseIcon onClick={handleClose} />
+      ) : (
+        <HamburgerIcon onClick={handleOpen} />
+      )}
+
+      <NavLinks className={isOpen ? 'open' : null}>
+        <LinkItem>
+          <NavigationLink to="/books" onClick={handleClose}>
+            Books
+          </NavigationLink>
+          <NavigationLink to="/categories" onClick={handleClose}>
+            CATEGORIES
+          </NavigationLink>
+        </LinkItem>
+      </NavLinks>
+      <UserIcon type="icon" />
+    </Nav>
   );
-}
+};
+
+export default Navbar;
+
+const Nav = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1rem;
+  height: 50px;
+  font-family: var(--montserrat);
+  @media screen and (min-width: 768px) {
+    height: 80px;
+  }
+`;
```

Comment:
- [ ] I would suggest you stick as much as possible to the Figma design. Book store CMC and logo at the right should be aligned with the added book. See the image below:
### Your design
![your](https://user-images.githubusercontent.com/81727988/180430065-4148d034-174f-4058-9960-68a8ad2e3d84.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/180430100-69f0462f-143f-409c-b9f2-62ab8a502250.png)